RELEASE_NOTES.rst

```diff
@@ -1,3 +1,278 @@
+coala 0.12 - XBear
+==================
+
+This release was brought to you by 116 contributors contributing 353
+commits.
+
+This release was long overdue, and contains several improvements
+in many components of coala, as well as the latest GSoC projects
+and features.
+
+The Next-Gen core is still on its way, but it's nearly done, and you can
+soon enjoy coala's new core features :)
+
+**Breaking Changes**
+
+- Write output to empty file instead of appending.
+  When ``--output`` option is specified, write output of coala to a
+  fresh file. This avoid file corruption with for certain output formats
```

Comment:
I think you intended to put a `/` between 'with' or 'for' here. Or use one of them('with' or 'for'). :sweat_smile:

Reply:
Oh indeed :+1:

Reply:
(I remember I've copied this from the commit messages, seems a few grammatical mistakes sneaked in ;) )